romana/kubernetes/conductor.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2016 Pani Networks
```

Comment:
well it is CLI but it also has adapters (e.g. openstack - that package contains code specific to talking to OpenStack). Wha do we want to do about it? @pritesh  ?